ethers-providers/src/rpc/transports/ws/mod.rs

```diff
@@ -50,6 +50,34 @@ impl WsClient {
         Ok(this)
     }
 
+    /// Establishes a new websocket connection. This method allows specifying a custom websocket
+    /// configuration.
```

Comment:
can we link to tungsenite's connect functions here in the docs?

Reply:
done